lib/ios.dart

```diff
@@ -185,27 +185,23 @@ void _applyImageiOS({
     print('$imagePath could not be loaded.');
     exit(1);
   }
-  for (final template in list) {
-    _saveImageiOS(template: template, image: image, targetPath: targetPath);
-  }
-}
 
-/// Saves splash screen image to the project
-void _saveImageiOS({
-  required _IosLaunchImageTemplate template,
-  required Image image,
-  required String targetPath,
-}) {
-  final newFile = copyResize(
-    image,
-    width: image.width * template.pixelDensity ~/ 4,
-    height: image.height * template.pixelDensity ~/ 4,
-    interpolation: Interpolation.cubic,
-  );
+  await Future.wait(
+    list.map(
+      (e) => Isolate.run(() async {
```

Comment:
I think this would read easier if you used the variable name `template` instead of `e`.